react-nesting/react-nesting_navigation/src/components/Image.jsx

```diff
@@ -0,0 +1,3 @@
+export default function Image({ src }, { alt }) {
```

Comment:
```suggestion
export default function Image({ src, alt }) {
```
As just discussed. Same applies to this component.